OMCompiler/Compiler/NBackEnd/Modules/3_Post/NBSolve.mo

```diff
@@ -693,43 +711,43 @@ protected
       // cref = exp
       case (Expression.CREF(cref = checkCref), exp)
         guard(ComponentRef.isEqual(cref, checkCref) and not Expression.containsCref(exp, cref))
-      then (eqn, Status.EXPLICIT, false);
+      then (eqn, Status.EXPLICIT, RelationInversion.FALSE);
 
       // 2. only swap lsh and rhs
       // exp = cref
       case (exp, Expression.CREF(cref = checkCref))
         guard(ComponentRef.isEqual(cref, checkCref) and not Expression.containsCref(exp, cref))
-      then (Equation.swapLHSandRHS(eqn), Status.EXPLICIT, true);
+      then (Equation.swapLHSandRHS(eqn), Status.EXPLICIT, RelationInversion.TRUE);
 
       // 3.1 negate (MINUS) lhs and rhs
       // -cref = exp
       case (Expression.UNARY(exp = Expression.CREF(cref = checkCref)), exp)
         guard(ComponentRef.isEqual(cref, checkCref) and not Expression.containsCref(exp, cref))
-      then (Equation.updateLHSandRHS(eqn, Expression.negate(lhs), Expression.negate(rhs)), Status.EXPLICIT, false);
+      then (Equation.updateLHSandRHS(eqn, Expression.negate(lhs), Expression.negate(rhs)), Status.EXPLICIT, RelationInversion.FALSE);
```

Comment:
If we negate both lhs and rhs but keep them on their respective side, the relation must be inverted, right?
```suggestion
      then (Equation.updateLHSandRHS(eqn, Expression.negate(lhs), Expression.negate(rhs)), Status.EXPLICIT, RelationInversion.TRUE);
```